server/main_test.go

```diff
@@ -135,6 +135,7 @@ func setupServer(mt *mainT) error {
 	config.LocalizationSettings.SetDefaults()
 	config.SqlSettings = *storetest.MakeSqlSettings("postgres", false)
 	config.ServiceSettings.SiteURL = model.NewString("http://example.com/")
+	config.ServiceSettings.AllowedUntrustedInternalConnections = model.NewString("localhost")
```

Comment:
This allows MM image proxy to pull the gif from the plugin

Reply:
See above, but if true this might be an issue for installations where the URL changes as well, so we should use a relative path instead.